pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: flutter_linkify
-description: Turns text URLs into clickable inline links in text for Flutter.
-version: 1.1.1
+description: Turns text URLs and Mailto links into clickable inline links in text for Flutter.
+version: 1.2.0
```

```suggestion
version: 2.0.0
```

To match the breaking change above.

We could support both onOpen and onOpenLink (and deprecate onOpen). To to you to make the decision!